css/templates/e-commerce/page.css

```diff
@@ -557,3 +557,32 @@ amp-selector [option][selected] {
     margin-left: calc(1/12 * 100%);
   }
 }
+
+
+/* CART */
+.commerce-cart-icon {
+  color: var(--color-font-accent);
+}
+
+.commerce-cart-total {
+  border-top: 1px solid var(--color-separator);
+  border-bottom: 1px solid var(--color-separator);
+}
+
+@media (--breakpoint-md) {
+  .commerce-cart-item {
+    border-bottom: 1px solid var(--color-separator);
+  }
+
+  .commerce-cart-button {
+    text-align: left;
+  }
+
+  .commerce-cart-item-image {
+    max-width: 100px;
```

Comment:
Prevent image from getting larger than 100px on wider screens since this is inside of a `col-2`. 

The only other way I could think of doing this was to have two `amp-img`, one with fixed layout and another with responsive and then show/hide dependant on the device width. I did't do this approach since it feels quite hacky.